bin/bump_version.py

```diff
@@ -56,7 +59,7 @@ def bump_version():
         print('error: Uncommitted changes detected.')
         sys.exit(1)
 
-    print(              'Current version:', current_version)  # noqa
+    print('Current version:', current_version)  # noqa
```

Comment:
So, black ignores `# noqa`?

(I don't mind this change, btw; it does look funky, but someone seems to have intentionally written this? So I don't like any automatic thing overriding the intentional style decisions made by a human. (At least not until we have a few more AI breakthroughs.))

Reply:
I think that it was totally a mistake? It doesn't line up with anything else...

And yes, black doesn't care about noqa, that's for flake8. It cares about style: on / style: off.

Reply:
The colon lines up with the colon of `'    New version: '`. Not saying I'm a fan, btw, but someone care enough to do this and add `# noqa`.

> It cares about style: on / style: off.

Hmmm, that's quite heavy :-/

Reply:
documentation tolds tkat black respect yapfdirectives https://black.readthedocs.io/en/stable/the_black_code_style.html?highlight=fmt%20on%20#code-style and yapf has one line disable https://github.com/google/yapf#why-does-yapf-destroy-my-awesome-formatting with `#  yapf: disable`. You may check

Reply:
Black now supports single-line ignores, `# fmt: skip`.